roadmap/EF2023.md

```diff
@@ -0,0 +1,142 @@
+the roadmap we have submitted internally within the EF.
+
+Things in the Ethereum ecosystem move quickly, so please note that we will likely deviate
+substantially from this initial roadmap throughout the year.
+
+See our "live" [roadmap](./README.md) document for an up-to-date overview on what we
+are currently up to.
+
+## Tracks
+
+The roadmap is divided into the following "tracks", identifying our main focus areas
+for the year.
+
+### DevEx & VM
+
+The [EthereumJS](https://github.com/ethereumjs/ethereumjs-monorepo) libraries are often used in a development context, serving e.g. as an integral part of main Ethereum developer tools or as building blocks for browser UIs of decentralized applications (dApps). Improving the experience when working with our libraries is therefore a crucial part of our work, with the VM playing a special role here due to the central place it takes in development tools like Hardhat, Truffle or Remix.
+
+### Hardforks & EIPs
+
+Implementing hardforks and providing early-on access to the added or changed functionality in development tools takes a reoccurring share in our work. We also do selected implementations of dedicated EIPs not yet "hardfork-settled" where it makes sense and help on improving EIP specifications and tests.
+
+### Research, Client & Testnets
+
+The JavaScript team takes part in Ethereum protocol-related research especially in areas where possible future applications in a TypeScript/JavaScript/Web Browser context are on the horizon, going "stateless" is a good example here since it is desirable to have lightweight browser access to the Ethereum network. We use our EthereumJS [client implementation](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/client) here as a research leveraging tool and for setting up fast-iterating functional testnets for early-on EIP testing in a multi-client context.
+
+### Portal Network
+
+The [Portal Network](https://github.com/ethereum/portal-network-specs) is a new peer-to-peer network topology for the Ethereum network (like devp2p) being built to allow for lightweight Ethereum (mainnet) protocol access and expose a JSON-RPC like interface along. The effort is led by [Piper Merriam](https://github.com/pipermerriam) and progress has been substantially advanced with specs laid out on a detailed level and two client implementations (Trin [Rust] and Fluffy [Nim]) underway.
+
+The EF JavaScript team has built a [prototype](https://github.com/acolytec3/portalnetwork) of this in 2021 to test its feasibility. In 2022 this became a steady project with a dedicated team and a full-featured JS-based Portal Network client is now developed within the realm of the [Ultralight](https://github.com/ethereumjs/ultralight) monorepo. 2023 will be the year where the Portal Network will get closer to a production state and hopefully early-on integrations with dAppps and/or Wallet providers can take place to test a wider roll-out of the new network topology.
+
```

Comment:
It would be cool to integrate the History Network in production somewhere if we can find a candidate.  EthOS might be an interesting exploration since they have a light client embedded directly into the Android OS and could be fun to supplement that with Ultralight (or provide as an alternative installation)

Reply:
Have added:

```md
- History Network Reference Production Integration (e.g. EthOS, other) Analysis
```

to Q3 and...

```md
- History Network Reference Production Integration Partnership (based on Q3 analysis)
```

to Q4.